src/fontra/views/fontoverview/fontoverview.js

```diff
@@ -50,6 +63,77 @@ export class FontOverviewController extends ViewController {
     );
   }
 
+  getFileMenuItems() {
```

Comment:
Why isn't this code in fontra-menus.js? Seems like the editor and font info should have this in common.

Reply:
Good point. I moved it to fontra-menus.js. I had applications settings in mind, where it might not be used. But in this case, we can skip the 'File' menu item.